rental.rb

```diff
@@ -1,10 +1,10 @@
 class Rental
-  attr_accessor :person, :book, :date
+  attr_accessor :date, :book, :person
 
   def initialize(date, book, person)
     @date = date
-    @person = book
-    @book = person
+    @book = book
+    @person = person
     book.rentals << self
     person.rentals << self
   end
```

Comment:
- [ ] Good job so far in implementing the changes requested by the previous reviewers, :clap: , however, as mentioned by the previous reviewer, you code throws errors when I enter the date, Please resolve it. :slightly_smiling_face: 

Below is the screenshot of how it behaves: :arrow_down: 

![Student Rentals](https://github.com/melkamu12/OOP-school-library/assets/43528594/8509e132-9aa1-434c-810d-2fa34ffe13af)

Reply:
-   Select a book from the list by number not other
- Select a person from list by nuber(not Id) it is not associated with other the date error for me it works fine 
https://drive.google.com/file/d/1yYkIFqNQd5gKrklh3oAG5uK6K08mRNfq/view?usp=sharing